Source/Blazorise/Fluent/FluentColumn.cs

```diff
@@ -191,14 +192,16 @@ public class FluentColumn :
 
     private class ColumnDefinition
     {
+        public ColumnWidth ColumnWidth { get; set; }
+
         public Breakpoint Breakpoint { get; set; }
 
         public bool Offset { get; set; }
     }
 
-    private ColumnDefinition currentColumn;
+    private ColumnDefinition columnDefinition;
```

Comment:
When looking at the class and at what this was doing, without even looking at the changes made, I got the feeling this should be named "current" since this can change according to the fluent context. Right? 
`private ColumnDefinition columnDefinition;` to `private ColumnDefinition currentColumnDefinition;`

Reply:
done